### PR TITLE
Change RDNN

### LINKED ISSUE #4: generated Metainfo contains deprecated developer_name tag
should be `developer` https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer

### PR DESCRIPTION
**Pull request by alainm23**
Fixed #4

### CODE REVIEW

**Comment by dar5hak** [comment]
It might be a good idea to use a hyphen (`ellie-commons`) instead of an underscore, because `app-generator` has a hyphen, which is more common anyway.

Additionally, you'll need to rename the pot file, the screenshot file, etc.

**Comment by dar5hak** [comment]
Ah, my bad.

File: data/app-generator.appdata.xml.in [hunk 1]
```diff
@@ -12,17 +12,21 @@
...
 	</screenshots>
 	<kudos>
 		<kudo>ModernToolkit</kudo>
 		<kudo>HiDpiIcon</kudo>
 	</kudos>
-	<developer_name translate="no">elementary Community</developer_name>
+
+	<developer id="io.github.ellie-commons">
+    	<name>elementary Community</name>
+  	</developer>
```
Comment (ryonakano) [inline data/app-generator.appdata.xml.in:25]:
```suggestion
	<developer id="io.github.ellie-commons">
    	<name>elementary Community</name>
 </developer>
```

Broken indentation

File: data/app-generator.appdata.xml.in [hunk 2]
```diff
@@ -12,17 +12,21 @@
 	</description>
 	<screenshots>
 		<screenshot type="default">
-			<image>https://raw.githubusercontent.com/elementary-community/app-generator/refs/heads/main/data/io.github.ecommunity.app-generator.png</image>
+			<image>https://raw.githubusercontent.com/elementary-community/app-generator/refs/heads/main/data/io.github.ellie_commons.app-generator.png</image>
```
Comment (ryonakano) [inline data/app-generator.appdata.xml.in:15]: Do we rename the URL in another PR? (it would be automatically redirected thanks to GitHub though)

File: data/app-generator.appdata.xml.in [hunk 3]
```diff
@@ -12,17 +12,21 @@
...
 		</screenshot>
 	</screenshots>
 	<kudos>
 		<kudo>ModernToolkit</kudo>
 		<kudo>HiDpiIcon</kudo>
 	</kudos>
-	<developer_name translate="no">elementary Community</developer_name>
+
+	<developer id="io.github.ellie-commons">
```
Comment (ryonakano) [inline data/app-generator.appdata.xml.in:23]: Is there any reason we use `ellie-commons` only here, not `ellie_commons`?

File: data/app-generator.appdata.xml.in [hunk 4]
```diff
@@ -1,37 +1,39 @@
...
-		<value key="x-appcenter-color-primary">#603461</value>
-	</custom>
+    <id>io.github.ellie_commons.app-generator</id>
+    <metadata_license>CC-BY-4.0</metadata_license>
+    <project_license>GPL-3.0-or-later</project_license>
+    <name>App Generator</name>
+    <summary>Create an elementary OS app using one of the pre-made app templates</summary>
+    <description>
+        <p>
+              App Generator streamlines the process of creating Vala and GTK projects, allowing you to quickly get started without having to configure everything from scratch. With just a few clicks, it generates a well-organized and functional project structure, optimizing your workflow and saving you from repetitive setup tasks. Perfect for developers who want to focus on coding without worrying about configuration details.
```
Comment (ryonakano) [inline data/app-generator.appdata.xml.in:10]:
```suggestion
            App Generator streamlines the process of creating Vala and GTK projects, allowing you to quickly get started without having to configure everything from scratch. With just a few clicks, it generates a well-organized and functional project structure, optimizing your workflow and saving you from repetitive setup tasks. Perfect for developers who want to focus on coding without worrying about configuration details.
```

**Comment by ryonakano** [comment]
Maybe we may want to replace the last occurrence of `elementary-community` while we're here

```
[ryo@b760m ~/work/ellie-commons/app-generator (feat/RDNN =)]$ git grepn elementary-community
README.md:9:![Screenshot](https://raw.githubusercontent.com/elementary-community/app-generator/refs/heads/main/data/io.github.ellie_commons.app-generator.png)
[ryo@b760m ~/work/ellie-commons/app-generator (feat/RDNN =)]$
```

File: README.md [hunk 5]
```diff
@@ -6,7 +6,7 @@
   <h3 align="center">Create an elementary OS app using one of the pre-made app templates</h3>
 </div>
 
-![Screenshot](https://raw.githubusercontent.com/elementary-community/app-generator/refs/heads/main/data/io.github.ecommunity.app-generator.png)
+![Screenshot](https://raw.githubusercontent.com/elementary-community/app-generator/refs/heads/main/data/io.github.ellie_commons.app-generator.png)
```
Comment (ryonakano) [inline README.md:9]:
```suggestion
![Screenshot](https://raw.githubusercontent.com/ellie-commons/app-generator/refs/heads/main/data/io.github.ellie_commons.app-generator.png)
```

Ah, it's here.

**Review by ryonakano — APPROVED** [review]
LGTM, thank you for this!

**Comment by alainm23** [comment]
Thank you very much for your time
